Assets/ExpandedStationA/ExpandStationScene.unity

```diff
@@ -3755,17 +3755,17 @@ PrefabInstance:
     - target: {fileID: 1972979662120986136, guid: a06f1a9e44fccb3409ddc94a08e3cac6,
         type: 3}
       propertyPath: m_LocalPosition.x
-      value: 201.82
```

Comment:
Moved player into the office as it was bloody annoying having to run there every time